README.md

```diff
@@ -84,12 +84,20 @@ That person, is you.
 ### How to run the test suite
   * Clone the My Solar Garden FE repo
     * My Solar Garden FE connects to the Back End Repository via the the [Backend Heroku App](https://solar-garden-be.herokuapp.com/)
-  * In the terminal while ```CD'd``` into the FE repo, run ```bundle exec rspec```. This will run all of the tests within the FE repository.
+  * In the terminal while ```CD'd``` into the FE repo, run ```bundle exec rspec```. This will run all of the tests within the FE r
```

Comment:
Should this read as: 
```suggestion
  * In the terminal while ```CD'd``` into the FE repo, run ```bundle exec rspec```. This will run all of the tests within the FE repository
```